src/components/attendee/InputSelect/InputSelect.module.scss

```
@@ -16,9 +20,9 @@
   padding: space.empty(0.33) 5em space.empty(0.33) space.empty(1);
   border-radius: space.empty(0.33) 0px 0px space.empty(0.33);
   box-sizing: border-box;
-  width: 80%;
+  width: 100%;
   border: none;
-  border-right: 1px solid theme.$c-glass-panel-background;
+  border-right: border.$radius--s3 solid theme.$c-glass-panel-background;
```

`radius` is not `width`

```suggestion
  border-right: border.$width--slim solid theme.$c-glass-panel-background;
```

---

[a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)